components/pagination/src/pagination.ts

```diff
@@ -102,18 +103,131 @@ export class DtPagination implements OnInit {
   }
   private _currentPage = 1;
 
+  /**
+   * Aria label for the previous page button. Defaults to "Previous page"
+   * @deprecated `aria-label-previous` does not conform with accessibility standards.
+   * Please use `ariaLabelPrevious` input instead.
+   * @breaking-change Will be removed in version 7.0.0
+   */
+  @Input('aria-label-previous')
+  get ariaPreviousLabel(): string {
+    return this._ariaLabelPrevious;
+  }
+  set ariaPreviousLabel(value: string) {
+    this._ariaLabelPrevious = value;
+  }
   /** Aria label for the previous page button. Defaults to "Previous page" */
-  @Input('aria-label-previous') ariaPreviousLabel = ARIA_DEFAULT_PREVIOUS_LABEL;
+  @Input()
+  get ariaLabelPrevious(): string {
+    return this._ariaLabelPrevious;
+  }
+  set ariaLabelPrevious(value: string) {
+    this._ariaLabelPrevious = value;
+  }
+  /** @internal Aria label for the previous page button. Defaults to "Previous page" */
+  _ariaLabelPrevious = ARIA_DEFAULT_PREVIOUS_LABEL;
+
+  /**
+   * Aria label for the next page button. Defaults to "Next page"
+   * @deprecated `aria-label-next` does not conform with accessibility standards.
+   * Please use `ariaLabelNext` input instead.
+   * @breaking-change Will be removed in version 7.0.0
+   */
+  @Input('aria-label-next')
+  get ariaNextLabel(): string {
+    return this._ariaLabelNext;
+  }
+  set ariaNextLabel(value: string) {
+    this._ariaLabelNext = value;
+  }
   /** Aria label for the next page button. Defaults to "Next page" */
-  @Input('aria-label-next') ariaNextLabel = ARIA_DEFAULT_NEXT_LABEL;
+  @Input()
+  get ariaLabelNext(): string {
+    return this._ariaLabelNext;
+  }
+  set ariaLabelNext(value: string) {
+    this._ariaLabelNext = value;
+  }
+  /** @internal Aria label for the next page button. Defaults to "Next page" */
+  _ariaLabelNext = ARIA_DEFAULT_NEXT_LABEL;
+
   /** Aria label for the pagination. Defaults to "Pagination" */
   @Input('aria-label') ariaLabel = ARIA_DEFAULT_LABEL;
+
+  /**
+   * Aria label for the ellipsis character. Defaults to "The next pages are ellipses"
+   * @deprecated `aria-label-ellipses` does not conform with accessibility standards.
+   * Please use `ariaLabelEllipses` input instead.
+   * @breaking-change Will be removed in version 7.0.0
+   */
+  @Input('aria-label-ellipses')
+  get depAriaLabelEllipses(): string {
+    return this._ariaLabelEllipses;
+  }
+  set depAriaLabelEllipses(value: string) {
+    this._ariaLabelEllipses = value;
+  }
   /** Aria label for the ellipsis character. Defaults to "The next pages are ellipses" */
-  @Input('aria-label-ellipses') ariaLabelEllipses = ARIA_DEFAULT_ELLIPSES;
+  @Input()
+  get ariaLabelEllipses(): string {
```

Comment:
I think this should be named `ariaLabelEllipsis` instead (as you already wrote in the comments), as "ellipses" is plural.